### ISSUE TITLE
SSHKey update  Insufficient access

### ISSUE THREAD

**Issue by edperry** [ltb-project/self-service-password#105]
When using "user" for the who_change_sshkey  even though I have Delegated the authority for "User"  to be able to read and write this sshPubKey.   But I still get the following error.    

 It works fine when set to manager (which is an admin account ) as expected

I was wondering if anyone has a "Guide" / "HowTo" to properly delegate this authority . Eveytime I google for it, it comes up with Password delegations but that is a simple process and the sshPubKey is a custom one

```
[Sat Apr 15 20:47:33.407763 2017] [:error] [pid 17100] [client 10.64.145.81:53650] LDAP - Modify sshPublicKeys error 50 (Insufficient access), referer: https://reset.digital.hbc.com/?action=changesshkey
[Sat Apr 15 20:53:20.775293 2017] [:error] [pid 28346] [client 10.64.145.81:53790] PHP Warning:  ldap_get_values(): Cannot get the value(s) of attribute Decoding error in /var/www/reset.digital.hbc.com/pages/changesshkey.php on line 120, referer: https://reset.digital.hbc.com/?action=changesshkey
[``

**Comment by coudot** [comment]
Note that this feature has just been contributed and is not available in the stable version (1.0). We will test this feature a little more for 1.1 version.

**Comment by edperry** [comment]
I think this issue is more of an issue with our LDAP, some users got the delegation and others did not. So I am going to close this issue as I am sure it is working fine and my AD is not
